packages/datadog-plugin-jest/test/index.spec.js

```diff
@@ -452,6 +461,34 @@ describe('Plugin', () => {
         passingTestEvent.test.fn()
       })
 
+      it('works with http integration', (done) => {
```

Comment:
Is this what is actually tested? Or is it testing context propagation more generally?

Reply:
Also, if it is actually testing propagation more generally, HTTP is not needed which would avoid having to use Nock.

Reply:
yeah, this is the actual test. 

Do you mean I could use a different library for testing propagation? 
The reason we've singled out http and database integrations is because 
1. they're common
2. http requests can create traces in different services, and it is important that the origin tag is propagated

Reply:
Makes sense for HTTP requests. But then what is additionally tested by #1410 ?

Reply:
#1410 is not ready for review yet. I wanted to give a more thorough description before moving it to ready for review. 

The choice of `pg` as a "database library" is arbitrary in #1410. The one thing that could be interesting in that PR is the ability to test "cross integration", that is, that if a library that we instrument calls another one, things like `origin` are propagated correctly. 
This is easy to test with `http` but was more difficult with `pg`, hence the `TEST_DEPENDENCIES` env var: https://github.com/DataDog/dd-trace-js/pull/1410/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R625. 
I'm not sure if we want to do that though. 

As I mentioned, I wanted to finish up this PR before moving to #1410

Reply:
`pg` wouldn't propagate since it would never have children.

Reply:
yeah that's a good point. We might not need this

Reply:
and closed :)